examples/howto/marginalizing-models.ipynb

```diff
@@ -0,0 +1,1440 @@
+{
```

Comment:
<p>Plot the switchpoint variable (and lp), which would be directly comparable to the Stan article?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/628/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>